internal/addrs/self.go

```diff
@@ -1,14 +0,0 @@
-package addrs
```

Comment:
Why is this file getting removed in this PR?

Reply:
I just wanted to sneak it in... its dead code, I know not the best habit, it somehow was missed in my formal cleanup PR.